test/platform/item/emote/adr-95/representation.spec.ts

```diff
@@ -0,0 +1,68 @@
+import expect from 'expect'
+import {
+  BodyShape,
+  EmoteRepresentationADR73
+} from '../../../../../src/platform/item'
+import { testTypeSignature } from '../../../../test-utils'
+
+describe('Emote representation tests', () => {
+  const representation: EmoteRepresentationADR73 = {
+    bodyShapes: [BodyShape.FEMALE],
+    mainFile: 'file1',
+    contents: ['file1', 'file2']
+  }
+
+  testTypeSignature(EmoteRepresentationADR73, representation)
+
+  it('static tests must pass', () => {
+    expect(EmoteRepresentationADR73.validate(representation)).toEqual(true)
+    expect(EmoteRepresentationADR73.validate(null)).toEqual(false)
+    expect(EmoteRepresentationADR73.validate({})).toEqual(false)
+  })
+
+  it('representation without body shape fails', () => {
```

Comment:
Same here, for all the validations returning false, I'd add a check for the array of errors returned.